### DISCUSSION TITLE
Spotconnect in Home Assistant

### DISCUSSION THREAD

**Discussion by L3nnoX** [philippe44/SpotConnect#47] [Q&A]
After searching for quite some time, I stumbled across your project what does exactly what I need. The only problem is, that I need it in Home Assistant (running on a Raspi 5). Is there a chance that you can adapt it, or is there another workaround for the problem?
Music Assistant is not working for my usecase, because I want to control everything via the Spotify app and stream the music to my Yamaha RX-V473 which can use Airplay or DLNA as input. I also tried Spotify connect as an Addon, which gives me no way of an output to DLNA (it outputs the music locally on the Raspi??)
Perhaps someone has a cool idea for this usecase. Thanks in advance!

**Comment by philippe44** [comment]
I think the guy who adapted my AirConnect to HA also did SpotConnect

**Comment by rmflom** [comment]
Was there ever a solution for this?
I'm trying to implement this, but havent really found a way to properly implement this into Home Assistant...

**Reply by L3nnoX** [reply]
Unfortunately not...
I now bought a WiiM Ultra Streamer :(

**Reply by philippe44** [reply]
You should contact who did the airconnect integration. My 2 apps are very similar.
